applications/crossbar/src/modules/cb_rates.erl

```diff
@@ -133,6 +150,23 @@ put(Context) ->
 delete(Context, _RateId) ->
     crossbar_doc:delete(Context).
 
+-spec validate_number(ne_binary(), cb_context:context()) -> cb_context:context().
+validate_number(Phonenumber, Context) ->
+    case length(re:replace(Phonenumber, "[^0-9]", "", [global, {return, list}])) >= 7 of 
```

Comment:
May be it is time to change default to something like this: <<"^\+?(\d{10,})$">>
I use it for long time without complaints :)